### DISCUSSION TITLE
Welcome to discordrb Discussions!

### DISCUSSION THREAD

**Discussion by Daniel-Worrall** [shardlab/discordrb#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as an alternative to [the discord](https://discord.gg/cyK3Hjm) to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by greenbigfrog** [comment]
shitposter, ex-ruby user. now using the superior option: crystal
(also, somehow managed to earn myself the Proficient role for drb in dAPI)
